packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/bitmap.dart

```diff
@@ -5,6 +5,7 @@
 import 'dart:async' show Future;
 import 'dart:typed_data' show Uint8List;
 
+import 'package:flutter/cupertino.dart';
```

Comment:
Looks like your IDE chose a weird auto-import here; this should be a new `show` below instead.

Reply:
(Specifically, from `foundation.dart`)